adjust introduction of readme (#27)
asreview/asreview-insights: README.md

[![PyPI version](https://badge.fury.io/py/asreview-insights.svg)](https://badge.fury.io/py/asreview-insights) [![Downloads](https://pepy.tech/badge/asreview-insights)](https://pepy.tech/project/asreview-insights) ![PyPI - License](https://img.shields.io/pypi/l/asreview-insights) ![Deploy and release](https://github.com/asreview/asreview-insights/workflows/Deploy%20and%20release/badge.svg) ![Build status](https://github.com/asreview/asreview-insights/workflows/test-suite/badge.svg)

This official extension to [ASReview
LAB](https://github.com/asreview/asreview) extends the software with tools for

> [plotting](#plot-types) and extracting the [statistical results](statistical-results) 

Reviewer: ```suggestion
[plotting](#plot-types) and extracting the [statistical results](#metrics) 
```

of several [performance metrics](performance-metrics). The

> extension is especially useful in combination with the simulation mode[**NEEDS
> HYPERLINK TO RTDs**] of ASReview LAB.

Reviewer: ```suggestion
extension is especially useful in combination with the [simulation functionality](https://asreview.readthedocs.io/en/latest/simulation_overview.html) of ASReview LAB.
```

![Recall with multiple lines](docs/example_multiple_lines.png)

> ## Statistical results

Reviewer: I'm struggling with the wording here. The API uses the word "metrics". I decided to use metrics internally because we are talking about metrics and scores, not statistics. This keeps the API and wording in line with the wording used machine learning packages like sklearn and tensorflow (we depend on both packages). 

However, the CLI uses "stats", which is legacy. I was thinking about renaming this. This is actually the best moment for renaming. I don't see any stats we want to include here in the nearby future, as dataset statistics are moved to datatools extension.

Author: Yes, I agree. But be aware that metrics are the wss, erf and recall. Their time series results are presented in a plot (via the plotting command) or as point estimates (via the stats command). Actually, stats should be used for computing statistics, like the mean wss or the variance of erf across mutiple simulation runs. 

I propose to change stats to metrics in the CLI (we can add stats in a later stage if we add functionality  for computing means/variance of metrics) and, in the readme, refer to the point estimates as numerical results, output or something similar.

> ## Statistical results

Reviewer: I'm struggling with the wording here. The API uses the word "metrics". I decided to use metrics internally because we are talking about metrics and scores, not statistics. This keeps the API and wording in line with the wording used machine learning packages like sklearn and tensorflow (we depend on both packages). 

However, the CLI uses "stats", which is legacy. I was thinking about renaming this. This is actually the best moment for renaming. I don't see any stats we want to include here in the nearby future, as dataset statistics are moved to datatools extension.

> ## Statistical results

Reviewer: ```suggestion
## Metrics
```
